test.js

```diff
@@ -20,6 +20,13 @@ test('valid SVGs', t => {
 <svg version="1.1">
 </svg>
 `));
+	t.true(isSvg(`<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+xmlns="http://www.w3.org/2000/svg"
+width="75"
+height="75"
+version="1.1"
+></svg>`));
```

Comment:
This test passes on the `main` branch.

Reply:
I just tried it again, does not work for me. I also tried it on runkit, still does not work.

https://runkit.com/embed/r74bv6am4lp6